modules/apps/image-uploader/image-uploader-web/src/main/resources/META-INF/resources/upload_image.jsp

```diff
@@ -111,7 +111,7 @@ String tempImageFileName = ParamUtil.getString(request, "tempImageFileName");
 									</aui:validator>
 
 									<aui:validator name="maxFileSize">
-										'<%= String.valueOf(maxFileSize) %>'
+										'<%= LanguageUtil.formatStorageSize(maxFileSize, locale) %>'
```

Comment:
@dvhoang2106 
Please follow this example and update your source code
`<aui:validator errorMessage='<%= LanguageUtil.format(locale, "please-enter-a-file-with-a-valid-file-size-no-larger-than-x", LanguageUtil.formatStorageSize(dlEditFileEntryDisplayContext.getMaximumUploadSize(), locale)) %>' name="maxFileSize"><%= dlEditFileEntryDisplayContext.getMaximumUploadSize() %></aui:validator>`